src/components/DevDummyLogin/index.js

```diff
@@ -58,13 +58,13 @@ const S = StyleSheet.create({
   },
   divider: {
     width: '100%',
-    backgroundColor: COLORS.blackgrey,
+    backgroundColor: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.blackgrey` to `COLORS.gray400` for the divider's background color seems fine if it aligns with the new design requirements. However, ensure that this color change doesn't affect readability or accessibility (contrast) in the UI.



<!-- This is an auto-generated comment by OSS CodeRabbit -->